sdk/core/azure-core/vcpkg/CONTROL

```diff
@@ -7,17 +7,16 @@ Build-Depends: curl, nlohmann-json
 Description: Microsoft Azure Core SDK for C++
   This library provides shared primitives, abstractions, and helpers for modern Azure SDK client libraries written in the C++.
 Homepage: https://github.com/Azure/azure-sdk-for-cpp/tree/master/sdk/core/azure-core
-Supports: !uwp
 Default-Features: http
 
 Feature: http
-Build-Depends: azure-core-cpp[core, curl], azure-core-cpp[core, curl, winhttp] (windows)
+Build-Depends: azure-core-cpp[core, curl], azure-core-cpp[core, curl, winhttp] (windows&!uwp)
```

Comment:
So, if winhttp doesn't work for uwp, is the only solution for someone building the SDK on Windows to send HTTP requests, to use curl or their own http stack? We probably want to document that somewhere.

Reply:
WinHTTP supports `WINAPI_FAMILY_SYSTEM`, `WINAPI_FAMILY_SERVER`, `WINAPI_FAMILY_GAMES`, and `WINAPI_FAMILY_DESKTOP_APP` (see `winhttp.h`). When VcPkg builds for "uwp" configuration, it is `WINAPI_FAMILY_PC_APP`, which is none of the above. So, it is not even the case that WinHTTP is only supported on some of the UWP. It isn't supported on any. At least in VcPkg's definition of "uwp". If you think this is still a problem and needs to be documented more broadly, please create a work item or a PR. I am not sure what is the best place to document which platforms does WinHTTP support. But probably it wouldn't be a big surprise for vcpkg users who use `uwp` triplet - in their mind, uwp is `WINAPI_FAMILY_PC_APP`.